docs/Contribute-to-Documentation.md

```diff
@@ -20,26 +20,73 @@ You are also welcome to discuss the content and share your feedback and thoughts
 
 ### Setting up the environment
 
-1. Fork the Wiki [repository](https://github.com/AlmaLinux/wiki).
-2. Create a new branch.
-3. Edit or create a page.
-4. Commit your changes.
-5. Create a pull-request to the master branch as described in the GitHub [documentation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+The Wiki is powered by [VuePress](https://vuepress.vuejs.org/).
 
-The Wiki is powered by [VuePress](https://vuepress.vuejs.org/), so it's pretty
-easy to run it locally (run all commands from the project's root):
+The general process to contribute to the wiki includes these steps:
+* Fork the Wiki [repository](https://github.com/AlmaLinux/wiki).
+* Clone your forked repository and navigate to its directory.
+* Create a new branch.
+* Edit or create a page.
+* Check your changes on the local wiki - see the steps below.
+* Commit your changes.
+* Create a pull-request to the master branch as described in the GitHub [documentation](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
 
-1. Install necessary dependencies:
+There are two ways to run the wiki locally to check your changes:
+
+**Please, note, most steps are suitable for all three Linux, Windows and macOS systems, but some commands may need some adjustments.**
+
+#### Install yarn 
+
+:::tip
+For Windows and macOS, you may need to install [nodejs](https://nodejs.org/en) first.
+:::
+
+* Install necessary dependencies:
    ```sh
    $ yarn install
    ```
-2. Run a development web-server:
+* Run a development web-server:
    ```sh
    $ yarn docs:dev
    ```
-3. Your local Wiki instance should be up and running on
+* Your local Wiki instance should be up and running on
    [http://localhost:8080/](http://localhost:8080/).
 
+#### Use a container 
```

Comment:
Since the original steps don't properly work on Linux systems, @LKHN and I added the container approach. It requires Containerfile script to be added to the wiki so users can create container images and containers without creating this file themselves:
```
FROM docker.io/library/node:lts-alpine AS builder
LABEL stage=auto-clean-stage1
RUN mkdir -p /wiki/tmp
COPY . /wiki/tmp
WORKDIR /wiki/tmp
RUN yarn
FROM docker.io/library/node:lts-alpine
RUN mkdir /wiki
WORKDIR /wiki
COPY --from=builder /wiki/tmp/package.json /wiki/package.json
COPY --from=builder /wiki/tmp/node_modules /wiki/node_modules
COPY --from=builder /wiki/tmp/yarn.lock /wiki/yarn.lock
CMD yarn run docs:dev
```

@LKHN also suggested another option: to create an AlmaLinux wiki official image to shorten these steps and make it easier for users. 

@andrewlukoshko @bennyvasquez @jaboutboul @jonathanspw @codyro @xsub what do you think?

Reply:
I like the idea of adding a dev container as an option. If it's relatively easy to create and maintain, an official image that helps with wiki contributions is fine. I've got the commands working on my laptop, so I can confirm they work on MacOS. We can add Linux commands once we have a container, too, since it'll be essentially the same thing.